tests/transport/epics/pva/test_p4p.py

```diff
@@ -623,3 +627,46 @@ async def put_pvs():
         pytest.approx((coro_end_time - coro_start_time).total_seconds(), abs=0.05)
         == 0.1
     )
+
+
+def test_block_flag_waits_for_callback_completion():
+    class SomeController(Controller):
+        @command()
+        async def command_runs_for_a_while(self):
+            await asyncio.sleep(0.2)
+
+    controller = SomeController()
+    pv_prefix = str(uuid4())
+    fastcs = make_fastcs(pv_prefix, controller)
+    command_runs_for_a_while_times = []
+
+    async def put_pvs():
+        ctxt = Context("pva")
+        for block in ["true", "false"]:
+            start_time = datetime.now()
+            await ctxt.put(
+                f"{pv_prefix}:CommandRunsForAWhile",
+                True,
+                request=f"record[block={block}]",
+            )
```

Comment:
Looks like `ctxt.put(wait)` already does this:
https://github.com/bluesky/ophyd-async/pull/846

so we should be able to do:
```suggestion
        for block in [True, False]:
            start_time = datetime.now()
            await ctxt.put(
                f"{pv_prefix}:CommandRunsForAWhile",
                True,
                wait=block,
            )
```